TUTORIAL.md

```diff
@@ -213,17 +213,17 @@ let timerTick (dispatch: Dispatch<Msg>) =
 
-In other words, you don’t call the impure functions yourself; the MVU library calls them for you. Furthermore, from the point of view of your model, everything happens asynchronously (in the sense that your app and update loop continues without waiting on a response, and reacts to the “response” message when it arrives).
+In other words, unlike the model which is given to the view function for interpretation, the `Cmd<'msg>` function will be invoked by the MVU. From the point of view of your model, everything happens asynchronously: your app and UI update loop continues without waiting for the results of the `Cmds<'msg>` and when the commands complete the results are `dispatch` into the event loop as just another event.
 
 For example:
 
 * The user clicks a button to log in, which dispatches a `SignInRequested` message
-* The `update` function returns a new model with an `IsBusy = true` value (which can be to show an animation such as a spinner) as well as a command that actually calls the API
-* The MVU loop calls the command
-* The app continues to work normally - the spinner spins because `IsBusy = true`, and any other messages are processed as they would normally be. Note that you are of course free to process messages differently based on the fact that `IsBusy = true`; for example, you may choose to ignore additional `SignInRequested` messages.
-* When the API call returns, the function that called the API dispatches a suitable message based on the result (e.g. `SignInSuccessful` or `SignInFailed`)
+* The `update` function returns a new `model` with an `IsBusy = true` value (which can be shown as an animation such as a spinner) as well as a `Cmd<'msg>` function that calls the API asynchronously to perform the request and process the response onced invoked with a `dispatch` function.
+* MVU uses new model with view to update the UI and invokes the command by passing it a `dispatch` function.
+* The app continues to work normally - the spinner spins because `IsBusy = true` and any other messages are processed as normally. Note that you are of course free to process messages differently based on the fact that `IsBusy = true`. For example, you may choose to ignore additional `SignInRequested` messages.
+* When the API call finally returns (as detected by `await` continuing or `Task` completion event), the command function continues by calling the `dispatch` function to insert its result into the MVU eventloop (e.g. `SignInSuccessful` or `SignInFailed`) upon which the `update` function immediately performs to produce a new `model * Cmd<'msg>` result.
```

Comment:
We have already mentioned much of this in the second point now. Rephrase: "The API call finally returns, and the function that called the API uses its `dispatch` argument to dispatch a suitable message (e.g. `SignInSuccessful` or `SignInFailed`)"